ui/src/app/components/workflows/workflow/dynamic-parts/cron-quartz-part/cron-quartz-part.component.html

```diff
@@ -30,11 +30,20 @@
 
 <!-- Free text input -->
   <clr-control-container *ngIf="inputType == InputTypes.FREE_TEXT">
-    <input  [readOnly]="false" clrControl [ngModel]="freeText" (ngModelChange)="onFreeTextChange($event)"/>
-
-    <input *ngIf="(validation | async) as validationResult; else loading" [readOnly]="true" clrControl
-           [value]="validationResult.isValid ? validationResult.explained : 'Incorrect expression!' "/>
+    <input clrControl
+           [name]="name+'-freetext-'+uiid"
+           (ngModelChange)="onFreeTextChange($event)"
+           [ngModel]="freeText"
+           [readOnly]="false"
+           [size]="100" />
```

Comment:
I didn't mean size 100 here. That makes no sense. Cron expression itself can not be longer than 13 characters (incl. whitespace)

Reply:
It can be for example "0,2,4,6,7,19,23,24,29,32,33,36,41,47,49,50,52,59 0 0 ? * * *"